.github/workflows/build.yml

```diff
@@ -268,46 +322,24 @@ jobs:
           aws-region: ${{ env.AWS_DEFAULT_REGION }}
           role-to-assume: ${{ secrets.TEST_ROLE_TO_ASSUME }}
           role-duration-seconds: 3600
-      # Disabling the unix-chkpwd AppArmor profile is necessary when
-      # running Molecule tests against Fedora 40 and 41; otherwise,
-      # the privileged container cannot successfully run sudo and
-      # hence Ansible is unable to do anything.  See
+      # AppArmor interferes when running Molecule tests against Fedora
+      # 40 and 41; it does not allow the privileged container to run
+      # sudo and hence Ansible is unable to do anything.  See
       # fedora-cloud/docker-brew-fedora#117 for more details.
       #
-      # Purging firefox is currently necessary because the
-      # installation available on the GitHub runner instance provides
-      # two conflicting AppArmor profiles:
-      # /etc/apparmor.d/usr.bin.firefox and /etc/apparmor.d/firefox.
-      # This conflict causes the aa-disable /usr/sbin/unix_chkpwd
-      # command to fail.
-      #
-      # Purging passt is currently necessary because the installation
-      # available on the GitHub runner instance contains a wonky
-      # AppArmor file (/etc/apparmor.d/abstractions/passt) that causes
-      # the aa-disable command to fail.
-      #
-      # TODO: Remove the apt-get purge and systemctl reload commands
-      # when possible.  See cisagov/skeleton-ansible-role#215 for more
-      # details.
-      - name: Disable unix-chkpwd AppArmor profile
-        run: |
-          sudo apt-get purge firefox passt
-          sudo systemctl reload apparmor.service
-          sudo apt-get install apparmor-utils
-          sudo aa-disable /usr/sbin/unix_chkpwd
-        if: ${{ startsWith(matrix.platform, 'fedora') }}
+      # There is a growing consensus that AppArmor causes too many
+      # problems and should not be active on the short-lived GitHub
+      # runners.  See, for example,
+      # actions/runner-images/issues/10015.  To avoid problems in the
+      # future, it makes sense to simply disable AppArmor altogether
+      # before running Molecule tests.
+      - name: Disable AppArmor
+        uses: cisagov/action-disable-apparmor@v1
       - name: Run molecule tests
         run: >-
-          molecule test
+          molecule -vvv test
```

Comment:
Did you mean to leave this commit (https://github.com/cisagov/ansible-role-cobalt-strike/pull/93/commits/b7a30bbdd9a09942368e3b79d95704f5db64fecb) in?

Reply:
No!  Thank you for catching that.  I dropped that commit.